wishlist.html

```diff
@@ -21,68 +22,111 @@
+              <a class="secondary-color" href="./index.html">
+                <li class="header-item">Home</li>
+              </a>
+              <a href="./productList.html">
+                <p class="header-item primary-color">Shop Now</p>
+              </a>
             </ul>
           </div>
-
+  
           <div class="search-bar-container search-bar-inline">
             <div class="search-bar">
-              <input
-                class="input-search"
-                type="text"
-                placeholder="Search your favorite brand and products"
-              />
+              <input class="input-search" type="text" placeholder="Search your favorite brand and products" />
               <span class="material-icons search-icon">
                 search
               </span>
             </div>
           </div>
-
-          <div class="icon-container flex">
+  
+          <div class="flex icon-container">
             <div class="icon-unit">
               <a class="flex-column flex-center secondary-color header-icon" href="./login.html">
-              <span class="material-icons icon"> login </span>
-              <p>Login</p></a>
+                <span class="material-icons icon"> login </span>
+                <p>Login</p>
+              </a>
             </div>
             <div class="icon-unit">
               <a class="flex-column flex-center secondary-color header-icon" href="./wishlist.html">
                 <span class="material-icons icon"> favorite </span>
-              <p>WishList</p></a>
+                <p>WishList</p>
+              </a>
             </div>
             <div class="icon-unit">
               <a class="flex-column flex-center secondary-color header-icon" href="./cart.html">
                 <span class="material-icons icon"> shopping_cart </span>
-              <p>Cart</p></a>
+                <p>Cart</p>
+              </a>
             </div>
           </div>
+          <span class="material-icons icon hide-menu menu-logo"> menu </span>
         </div>
-        <hr class="hr-line header-divider"/>
+        <hr class="hr-line header-divider" />
         <div class="search-bar-container search-bar-block">
           <div class="search-bar flex">
-            <input
-              class="input-search"
-              type="text"
-              placeholder="Search your favorite brand and products"
-            />
+            <input class="input-search" type="text" placeholder="Search your favorite brand and products" />
             <span class="material-icons search-icon flex-center"> search </span>
           </div>
         </div>
       </div>
+  
+      <div id="empty-box"></div>
+  
+      <nav class="collape-navbar-container hide-nav">
+        <div class="header-navbar flex justify-between">
+          <h2>magni<span class="secondary-color highlight-letter-header">Z</span>ent</h2>
+          <button class="nav-close-btn"><span class="material-icons icon"> close </span></button>
```

Comment:
do use code formatting and different tags on a different line, it seems messy to find where the tag is being closed here